src/components/styled-atlaskit/Button/Button.scss

```diff
@@ -17,6 +17,9 @@ button.styled-atlaskit-button {
     font-size: $font-size-h6;
     line-height: 30px;
     font-weight: normal;
+    span {
```

Comment:
can we take this out of button span selector? and instead declare it in new selector button > span > span?